cache/hhvm-scalable-cache.h

```diff
@@ -679,6 +684,11 @@ FastHashMonitor(){
   /* become aware of DC latency (hit lat) and DC miss latency + disk latency (miss lat) */
   PrintStepLat(DC_hit_lat, miss_lat);
 
+  if(exponentialBackoffRemaining > 0){
+    exponentialBackoffRemaining -= 1;
+    goto WAIT_STABLE;
```

Comment:
https://www.amazon.com/Coding-Standards-Rules-Guidelines-Practices/dp/0321113586

Reply:
This is an unproductive comment, please leave what the change should be.

Reply:
I'm not sure if you understood, but it was a joke of avoiding using goto, but obviously in this case it is valid and does not require a change